World.py

```diff
@@ -179,11 +181,13 @@ def __init__(self, id, settings):
         }
         max_tokens = 0
         if self.bridge == 'tokens':
-            max_tokens = self.bridge_tokens
+            max_tokens = max(max_tokens, self.bridge_tokens)
+        if self.lacs_condition == 'tokens':
+            max_tokens = max(max_tokens, self.lacs_tokens)
         tokens = [50, 40, 30, 20, 10]
         for t in tokens:
-            if t > max_tokens and f'{t} Gold Skulltula Reward' not in self.disabled_locations:
-                max_tokens = t
+            if f'{t} Gold Skulltula Reward' not in self.disabled_locations:
+                max_tokens = max(max_tokens, t)
```

Comment:
This doesn't look to me like a functional difference, except for checking each of the skulltula rewards prior to checking if `t > max_tokens` (in max). The other way used the `t > max_tokens` check as a short-circuit to skip the other check.

Reply:
I believe this was done for consistency with the other cases, but @Cuphat actually made these changes, so he's probably more fit to answer this.

Personally I don't mind either patterns, tho I'd argue consistency is better than a slight performance improvement in this case.

Reply:
Yeah, it was just consistency while I was changing all the others and adding Ganon BK from tokens. I don't have a strong opinion on that particular change.